src/lephare/__init__.py

```diff
@@ -32,6 +77,7 @@
 from ._pdf import *
 from ._photoz import *
 from ._spec import *
+from ._version import *
```

Comment:
Don't need to include this. It's automagical, and is only ever used by automated scripts, like automatic generation of documentation, etc.